modules/components/pages/outputs/azure_data_lake_gen2.adoc

```diff
@@ -0,0 +1,123 @@
+= azure_data_lake_gen2
+//tag::single-source[]
+:type: output
+:page-beta: true
+
+:categories: ["Services","Azure"]
+
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+
+Sends message parts as files to an https://learn.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-introduction[Azure Data Lake Gen2^] file system. Each file is uploaded with the file name specified in the `path` field.
```

Comment:
I've linked out to the Microsoft docs. But is this the best section to link to?

Reply:
That looks good!